tests/support/server.rs

```diff
@@ -54,11 +55,34 @@ pub fn spawn(txns: Vec<Txn>) -> Server {
                 thread::park_timeout(dur);
             }
 
+            // the server will encode the body as chunks if:
+            //   - a `write_timeout` has been supplied
+            //   - a `chunk_size` has been supplied
+            // 
+            // the server won't read headers so if the response doesn't specify `Transfer-Encoding`
+            // then it'll misbehave and send an invalid response.
```

Comment:
I don't think the server needs to do it's own chunked encoding, that can be part of the text passed to the macro, so that the exact text expected is always visible in the test cases.